bitbots_head_behavior/src/bitbots_head_behavior/actions/track_ball.py

```diff
@@ -23,5 +25,5 @@ def perform(self, reevaluate=False):
         # Get last ball position
         point = self.blackboard.world_model.get_ball_stamped()
 
-        # Call internal look-at to turn head to this point
-        self._look_at(point)
+        # Call internal look-at to turn head to this point (when necessary)
+        self._look_at(point, self.ball_tracking_min_pan_delta, self.ball_tracking_min_tilt_delta)
```

Comment:
With my previous comment you wouldn't need to handle this blackboard setting in *track_ball.py* but instead let *look_at.py* handle that on its own.

I would like that better because it respects seperation-of-concerns and is probably easier to service.